releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -0,0 +1,34 @@
+# spring-cloud-azure-feature-management & spring-cloud-azure-feature-management-web
+
+[Source code][source_code] | [Package (Maven)][package] | [Samples][samples]
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web]
+[Product documentation][docs]
```

Comment:
# spring-cloud-azure-feature-management

[Source code][source_code] | [Package (Maven)][package] | [Samples][samples] | [Product documentation][docs]
It's a little confusing. Do you like this?

# spring-cloud-azure-feature-management-web
[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]